export/http.go

```diff
@@ -0,0 +1,87 @@
+package export
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+
+	"go.opencensus.io/metric/metricdata"
+	"go.opencensus.io/resource"
+	"google.golang.org/protobuf/proto"
+)
+
+// HTTP is an exporter that exports metrics to an
+// HTTP endpoint
+type HTTP struct {
+	address   string
+	aPIkey    string
+	aPISecret string
+	headerMap map[string]string
+	client    *http.Client
+	config    Config
+}
+
+// NewHTTP returns a new exporter agent with an HTTP exporter attached
+func NewHTTP(address string, apikey string, apisecret string, headers map[string]string, config Config) *ExporterAgent {
+	headers["Content-Type"] = "application/x-protobuf"
+
+	exporter := HTTP{
+		address:   address,
+		aPIkey:    apikey,
+		aPISecret: apisecret,
```

Comment:
nit: would be good to keep the capitalization of parameters and configs the same...